src/pages/ProAmmPool/PositionListItem.tsx

```diff
@@ -286,6 +286,10 @@ function PositionListItem({
     )
   }, 0)
 
+  const totalFeeRewardUSD =
```

Comment:
do we need big number calculation: Fraction, ... ?

Reply:
Not quite. We use this style of calculation in a few more places